README.md

```diff
@@ -0,0 +1,83 @@
+# Unikraft Website
+
+Contents from this repository are published at https://unikraft.org/ with sub-links available from the top bar:
+
+- documentation: https://unikraft.org/docs/
+- community information (including hackathons): https://unikraft.org/community/
+- blog: https://unikraft.org/blog/
+
+The Unikraft website is written in [Markdown](https://www.markdownguide.org/) and published via [Hugo](https://gohugo.io/) and the [Docsy theme](https://www.docsy.dev/).
+
+## Build
+
+You can build the website inside a Docker environment.
+For this, you will need to install Docker-CE.
+You can do that by following the instructions [here](https://github.com/docker/docker-install), or simply run:
+
+**Note: This is highly recomended, since you will work in the same environment as the deployed website, so there will be no errors regarding packages version or filesystem layout.**
```

Comment:
```suggestion
**Note: This is highly recommended, since you will work in the same environment as the deployed website, so there will be no errors regarding packages version or filesystem layout.**
```